usmqe_tests/api/conftest.py

```diff
@@ -36,3 +36,32 @@ def invalid_session_credentials(request):
     invalid_token = request.param
     auth = TendrlAuth(token=invalid_token, username=username)
     return auth
+
+
+@pytest.fixture
+def cluster_reuse(valid_session_credentials):
```

Comment:
@fbalak All tests expect that there is one cluster which can be used for testing (we would save time and resources). If test needs cluster which is not there, it should be created and it is different test and ```cluster_create``` fixture should be used (it doesn't exist now).

I think that there is no reason to do here any big chunk of code to detect if there is any cluster and create it. It can lead to strange behaviour and it will waste our time.